tests/hooks/utils/mock-text-filter-fetch.js

```diff
@@ -0,0 +1,64 @@
+export const JAVA_RESULTS = [
```

Comment:
This module mocks API responses for text filter results based on query (or call to callFunction with specific args)